clients/src/test/java/org/apache/kafka/common/requests/RequestResponseTest.java

```diff
@@ -382,5 +385,31 @@ private AbstractRequestResponse createUpdateMetadataResponse() {
         return new UpdateMetadataResponse(Errors.NONE.code());
     }
 
+    private AbstractRequest createCreateTopicRequest() {
+        CreateTopicRequest.TopicDetails request1 = new CreateTopicRequest.TopicDetails(
+            3, 5, new HashMap<Integer, List<Integer>>(), new HashMap<String, String>());
 
+        Map<Integer, List<Integer>> replicaAssignments = new HashMap<Integer, List<Integer>>();
```

Comment:
To answer your other question, I had submitted a PR that removed all of these cases, but it didn't get merged in the end to avoid merge conflicts with some of the ongoing work for 0.9.0.0.